### PR TITLE
expand: add page

### PR DESCRIPTION
**Pull request by pxgamer**
----

- [x] The page (if new), does not already exist in the repo.

- [x] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited.

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines.

### CODE REVIEW

File: pages/windows/expand.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# expand
+
+> Uncompress one or more Windows Cabinet files.
+
+- Uncompress a single-file Cabinet file to the specified directory:
+
+`expand {{path/to/file.cab}} {{path/to/directory}}`
+
+- Display a list of files in the source Cabinet file:
```
Comment (agnivade) [inline pages/windows/expand.md]: Display _the_ list of files in _a_ source Cabinet file ?

File: pages/windows/expand.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+- Uncompress all files from the Cabinet file:
+
+`expand {{path/to/file.cab}} {{path/to/directory}} -f:*`
+
+- Uncompress a specific file from a Cabinet file:
+
+`expand {{path/to/file.cab}} {{path/to/directory}} -f:{{file}}`
+
+- Ignore the directory structure when uncompressing:
```
Comment (agnivade) [inline pages/windows/expand.md]: What does this result in ?

Comment (pxgamer) [reply pages/windows/expand.md]: It puts all files in the root of the output directory. In a flat structure.

File: pages/windows/expand.md (hunk 1 above)
Comment (pxgamer) [reply pages/windows/expand.md]: Yeah, that sounds better. 👍

File: pages/windows/expand.md (hunk 2 above)
Comment (pxgamer) [reply pages/windows/expand.md]: Perhaps this?
> Ignore the directory structure when uncompressing and add them to a single directory:

Comment (agnivade) [reply pages/windows/expand.md]: Just add a comma after "uncompressing".

**Pull request merged**